nupm/utils/dirs.nu

```diff
@@ -0,0 +1,39 @@
+# Directories and related utilities used in nupm
+
+# Decault installation path for nupm packages
+export const DEFAULT_NUPM_HOME = ($nu.default-config-dir | path join "nupm")
+
+# Prompt to create $env.NUPM_HOME if it does not exist and some sanity checks.
+export def nupm-home-prompt [] {
+    if 'NUPM_HOME' not-in $env {
+        error make {
```

Comment:
i would add a `--unspanned` here, otherwise, the error is less easy to read :confused: 
e.g. for the next `error make` as well
```
Error:   × Root directory ~/.config/nushell/config.nu exists, but
  │ is not adirectory. Make sure $env.NUPM_HOME points at a
  │ validdirectory and try again.
    ╭─[/home/amtoine/documents/repos/github.com/amtoine/nushell-nupm/nupm/utils/dirs.nu:15:1]
 15 │         if ($env.NUPM_HOME | path type) != 'dir' {
 16 │             error make {
    ·             ─────┬────
    ·                  ╰── originates from here
 17 │                 msg: ($"Root directory ($env.NUPM_HOME) exists, but is not a"
    ╰────
```